index.html

```diff
@@ -692,6 +714,16 @@ <h2><a href="#custom-canvas" id="custom-canvas" class="anchor">Custom Canvas</a>
           }
         }());
       },
+      stars: function stars() {
+        confetti({
+          particleCount: 100,
+          spread: 360,
+          ticks: 50,
+          gravity: 0,
+          shapes: ['star'],
```

Comment:
I like this demo, I think the colors are awesome! When I run it, it looks like the stars stop moving toward the end of the animation and just... freeze in place for a few ticks. Is that what you intended? I tweaked the `decay` and `startVelocity` a bit to smooth that out a little:

```suggestion
          decay: 0.94,
          startVelocity: 30,
          shapes: ['star'],
```

Let me know what you think